specification/batch/Azure.Batch/routes.tsp

```diff
@@ -140,8 +23,11 @@ alias Created<T> = {
 #suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
 #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Operation 'Exists' should be defined using a signature from the Azure.Core namespace."
 #suppress "@azure-tools/typespec-azure-core/rpc-operation-request-body" "BatchOperation with '@get' cannot have a body."
-@clientName("Batch")
-interface BatchApi {
+@client({
+  name: "BatchClient",
```

Comment:
If we rename the interface here to `BatchClient` do we need the name property here? (Doesn't seem like it does any harm, but I wonder if it's redundant)

Reply:
removed the name, doesn't appear to effect generation of the swagger or csharp sdk